### DISCUSSION TITLE
Sliding in  / out of the panel?

### DISCUSSION THREAD

**Discussion by simonchen** [brndnmtthws/conky#2020] [Ideas]
I would like to have a new feature that when mouse moving out of the panel , then the panel will be hidden as a very tiny slider tighting with screen's edge (either horizontal / vertical slider is fine), when mouse over the tiny slider then show up the panel again, i think the feature should be easily developed. thanks !

**Comment by Caellian** [comment]
I like this idea, but conky doesn't have support for animations atm. This single one could be hardcoded in, but a lot goes into making it behave nicely in edge cases (e.g. mouse enters while in collapsing transition), and in case more features like that are added in (e.g. expand size on hover) later, it would be a whole bunch of spagetti code without proper support for animations (transition trees to defer activation, separate variable update executor, variable clones to avoid jank, animation priorities, ...)
